tests/accel/upgrade/tests/upgrade.go

```diff
@@ -0,0 +1,177 @@
+package upgrade
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clusteroperator"
+	"github.com/openshift-kni/eco-goinfra/pkg/clusterversion"
+	"github.com/openshift-kni/eco-goinfra/pkg/namespace"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	"github.com/openshift-kni/eco-goinfra/pkg/route"
+	"github.com/openshift-kni/eco-gotests/tests/accel/internal/accelconfig"
+	accelinittools "github.com/openshift-kni/eco-gotests/tests/accel/internal/accelinittools"
```

Comment:
You can use dot import in order to avoid writing accelinittools prefix every time.
. "github.com/openshift-kni/eco-gotests/tests/accel/internal/accelinittools"